ts/Core/Axis/GridAxis.ts

```diff
@@ -910,38 +910,41 @@ class GridAxis {
         const gridOptions = options.grid || {};
         const userLabels = axis.userOptions.labels || {};
 
-        if (axis.horiz) {
-            if (gridOptions.enabled === true) {
-                axis.series.forEach(function (series): void {
-                    series.options.pointRange = 0;
-                });
-            }
+        // Fire this only for the Gantt type chart, #14868.
+        if (axis.options.grid && axis.options.grid.enabled) {
+            if (axis.horiz) {
+                if (gridOptions.enabled === true) {
```

Comment:
Now since we're already checking for `gridOptions.enabled` on line 914, we should be able to safely remove the check on line 916.